### PR TITLE
Better naming conventions

### PR DESCRIPTION
**Pull request by b1n-ch1kn**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimisation

## [required] Description

Lots of renaming of nodes (and a few topics) for better readability.
In future we should look at renaming all node scripts to be in the format `<description>_node` and have our source code format `node_<description>`. All source code files (nearly?) are already in this format, but the script format is inconsistent.

## [required] Documentation

Updated node names in `Driverless/Documentation/Software/ROS2 Packages and Nodes`

### [optional] Usability concerns or breaking changes?

If nodes aren't launching the way you would expect, look at their `setup.py` entry points. Have attempted to work in all of these changes.

### CODE REVIEW

File: src/hardware/steering_actuator/src/node_steering_actuator.cpp [hunk 1]
```diff
@@ -459,8 +459,8 @@ class SteeringActuator : public rclcpp::Node, public CanInterface {
                 double derivative_error = (error - this->prev_error) / elapsed_time_seconds;  // Grab derivative error
 
                 // left hand down is +, rhd is -
-                double target =
-                    -(Kp * error + Ki * this->integral_error + Kd * derivative_error);  // PID commands to send to plant
+                double target = -(this->Kp * error + this->Ki * this->integral_error +
+                                  this->Kd * derivative_error);  // PID commands to send to plant
```
Comment (Damin-G) [inline src/hardware/steering_actuator/src/node_steering_actuator.cpp:463]: going to assume this maths is still correct

File: src/navigation/py_slam/py_slam/node_imu_slam.py [hunk 2]
```diff
@@ -38,15 +38,15 @@ def wrap_to_pi(angle: float) -> float:  # in rads
     return (angle + pi) % (2 * pi) - pi
 
 
-class PySlam(Node):
+class IMUSlam(Node):
```
Comment (Damin-G) [inline src/navigation/py_slam/py_slam/node_imu_slam.py:41]: I have no idea why IMU stands for

File: src/navigation/py_slam/py_slam/node_wss_slam.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def wrap_to_pi(angle: float) -> float:  # in rads
     return (angle + pi) % (2 * pi) - pi
 
 
-class PySlam(Node):
+class WSSSlam(Node):
```
Comment (Damin-G) [inline src/navigation/py_slam/py_slam/node_wss_slam.py:40]: Dont know what WSS stands for either

**Review by Damin-G — APPROVED** [review]
Could be nice to create some small comments under the new node names with new acronyms for what they stand for

File: src/hardware/steering_actuator/src/node_steering_actuator.cpp (hunk 1 above)
Comment (b1n-ch1kn) [reply src/hardware/steering_actuator/src/node_steering_actuator.cpp:463]: yep, just pointing it explicitly to the class variable (if in the future we have other variables named the same)

File: src/navigation/py_slam/py_slam/node_imu_slam.py (hunk 2 above)
Comment (b1n-ch1kn) [reply src/navigation/py_slam/py_slam/node_imu_slam.py:41]: inertial measurement unit

File: src/navigation/py_slam/py_slam/node_wss_slam.py (hunk 3 above)
Comment (b1n-ch1kn) [reply src/navigation/py_slam/py_slam/node_wss_slam.py:40]: wheel speed sensor

**Pull request merged**